compose/frontend.yml

```diff
@@ -60,7 +60,7 @@ services:
     - ccd-network
 
   ccd-api-gateway:
-    image: "${CCD_API_GATEWAY_USE_LOCAL-hmctspublic.azurecr.io/}ccd/api-gateway-web:${CCD_API_GATEWAY_TAG:-latest}"
+    image: "hmctspublic.azurecr.io/ccd/ccd-api-gateway"
```

Comment:
why this change?

Reply:
This changes are only for our local development setup for testing and not intent to merge this PR.Sorry, for the confusion I have updated the PR description above.